letsencrypt/tests/renewer_test.py

```diff
@@ -0,0 +1,684 @@
+"""Tests for letsencrypt/renewer.py"""
+
+import configobj
+import datetime
+import mock
+import os
+import tempfile
+import pkg_resources
+import pytz
+import shutil
+import unittest
+
+ALL_FOUR = ("cert", "privkey", "chain", "fullchain")
+
+def unlink_all(rc_object):
+    """Unlink all four items associated with this RenewableCert.
+    (Helper function.)"""
+    for kind in ALL_FOUR:
+        os.unlink(rc_object.__getattribute__(kind))
+
+def fill_with_sample_data(rc_object):
+    """Put dummy data into all four files of this RenewableCert.
+    (Helper function.)"""
+    for kind in ALL_FOUR:
+        with open(rc_object.__getattribute__(kind), "w") as f:
+            f.write(kind)
+
+class RenewableCertTests(unittest.TestCase):
+    # pylint: disable=too-many-public-methods
+    """Tests for the RenewableCert class as well as other functions
+    within renewer.py."""
+    def setUp(self):
+        from letsencrypt import storage
```

Comment:
since you are testing `renewer.py`, this import should go at the top; likewise other same imports in this file